### ISSUE TITLE
JS decorator support

### ISSUE THREAD

**Issue by instrumaniak** [chinchang/web-maker#253]
Hi! I am getting errors while running js decorators (running mobx example codes) in web maker..in the babel mode....would you please check whether JS decorator is enabled for babel inside web maker? Thanks!

**Comment by marsch** [comment]
yeah think also that should be possible, think I've identified the line where to change the babel presets:

https://github.com/chinchang/web-maker/blob/master/src/script.js#L958-L960

`stage-2` needs to be changed to `stage-0` - but anyway would be cool to have a way to configure the babel-presets anyway - or have a bit more influence on how the code is transpiled in any case.
